.github/workflows/push-check.yml

```diff
@@ -0,0 +1,14 @@
+name: push-check
+on: [push]
+jobs:
+  push-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+      - name: Run tests
+        run: yarn test
```

Comment:
perché non provi anche a impostare due comandi per fare il check dei tipi con typescript e il lint.
Mi sembra ci sia anche in koajs_template